docs/GETTING-STARTED.md

```diff
@@ -0,0 +1,318 @@
+# Install kubernetes on Centos/RHEL/Fedora.
```

Comment:
Also, should this be renamed `GETTING-STARTED-RHEL-BASED.md` or something since it focuses on RHEL/CentOS/Fedora distros? Otherwise, a top-level `GETTING-STARTED.md` which references sub-level ones may make sense as well.

Reply:
Kyle, Thanks for your review.
I am making all of the above changes.
However, in my opinion it might be best to have all distros in one doc. Would you agree if I were to edit the document to put a generic top level, enter Ubuntu as a 2nd level heading but leave it a TODO for now.
And under rhel-rpm, I will enter subheadings for Fedora, then enter RHEL and CentOS but leave then a TODO as well.
I had done Fedora first because it is the easiest.

Reply:
@tfherbert Your suggestion of a single document is perfectly fine with me. Thanks!